fsspec/tests/test_config.py

```diff
@@ -14,7 +14,20 @@ def clean_conf():
     conf.clear()
 
 
-def test_from_env(clean_conf):
+def test_from_env_ignored(clean_conf):
+    env = {
+        "FSSPEC": "missing_proto",
+        "FSSPEC_": "missing_proto",
+        "FSSPEC__INVALID_KEY": "invalid_proto",
+        "FSSPEC_INVALID1": "not_json_dict",
+        "FSSPEC_INVALID2": '["not_json_dict"]',
+    }
+    cd = {}
+    set_conf_env(conf_dict=cd, envdict=env)
+    assert cd == {}
```

Comment:
I we make warnings, then we should assert that we have 5 raised here

Reply:
Done.